aicsimageio/readers/ome_tiff_reader.py

```diff
@@ -52,36 +48,19 @@ def _is_this_type(buffer: io.BufferedIOBase) -> bool:
             return True
         return False
 
-    @property
-    def data(self) -> np.ndarray:
-        if self._data is None:
-            # load the data
-            self._data = self.tiff.asarray()
-        return self._data
-
-    @property
-    def dims(self) -> str:
-        self._lazy_init_metadata()
-        dimension_order = self._metadata.image().Pixels.DimensionOrder
-        # reverse the string
-        dimension_order = dimension_order[::-1]
-        # see if t,z,or c is squeezed out.
-        # this is a tifffile implementation detail -- see squeeze_axes in tifffile.
-        if self.size_t() < 2:
-            dimension_order = dimension_order.replace("T", "")
-        if self.size_z() < 2:
-            dimension_order = dimension_order.replace("Z", "")
-        if self.size_c() < 2:
-            dimension_order = dimension_order.replace("C", "")
-        return dimension_order
-
     @property
     def metadata(self) -> omexml.OMEXML:
-        return self._lazy_init_metadata()
+        with TiffFile(self._file) as tiff:
+            if self._metadata is None and tiff.is_ome:
+                description = tiff.pages[0].description.strip()
+                if not (description.startswith("<?xml version=") and description.endswith("</OME>")):
+                    raise ValueError(f"Description does not conform to OME specification: {description[:100]}")
+                self._metadata = omexml.OMEXML(description)
 
-    def load_slice(self, slice_index=0):
-        data = self.tiff.asarray(key=slice_index)
-        return data
```

Comment:
@toloudis I removed this function because dask handles this now, but this may also be a breaking change. Should I add it back with a deprecated warning?

Reply:
I added this back to the TiffReader class which this inherits. Same implementation as there is no point to try and configure it to use the dask data.